robomaker-sample-app-ci/src/aws-robomaker-sample-application-ci.ts

```diff
@@ -70,12 +70,8 @@ async function fetchRosinstallDependencies(): Promise<string[]> {
   let packages: string[] = [];
   // Download dependencies not in apt if .rosinstall exists
   try {
-    for (let workspace of ["robot_ws", "simulation_ws"]) {
-      if (fs.existsSync(path.join(workspace, '.rosinstall'))) {
-        await exec.exec("rosws", ["update", "-t", workspace]);
-      }
-    }
     if (fs.existsSync(path.join(WORKSPACE_DIRECTORY, '.rosinstall'))) {
+      await exec.exec("vcs", ["import", "--input", ".rosinstall"], getExecOptions());
```

Comment:
Consider to define constant for reusable strings in multiple files.
e.g 
.rosinstall
import
--input

Reply:
Good catch, in this case I don't think it's necessary because there is only one file and one/two usage of the string. Defining constant in this case might be an overkill. 
We definitely should define constant for them when the repository expands.